content/documentation/pages/4-batch-developer-guides/2-batch/9-data-flow-simple-task-cloudfoundry.md

```diff
@@ -4,146 +4,151 @@ title: 'Deploying a task application on Cloud Foundry using Spring Cloud Data Fl
-      SPRING_CLOUD_DATAFLOW_APPLICATIONPROPERTIES_TASK_SPRING_DATASOURCE_HIKARI_MAXIMUMPOOLSIZE: 2
-    services:
-      - postgres-service
-```
-
-Some explanation of the configured properties:
-
-https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#\_common\_application_properties
-
-Now run `cf push -f ./manifest-dataflow.yml`.
-
-Once deployed, go to your Cloud Foundry dashboard. Both, Spring Cloud Skipper as well as Spring Cloud Data Flow should have a status of `Running`:
+This example uses Skipper, which you can set up on Cloud Foundry. To do so:
+
+1. Create a file named `manifest-dataflow.yml`:
+    ```yaml
+    ---
+    applications:
+      - name: data-flow-server
+        routes:
+          - route: <your-data-flow-server-route> # e.g. my-data-flow-server.cfapps.io
+        memory: 2G
+        disk_quota: 2G
+        instances: 1
+        path: ./spring-cloud-dataflow-server-2.1.0.M1.jar
+        env:
+          SPRING_APPLICATION_NAME: data-flow-server
+          SPRING_PROFILES_ACTIVE: cloud
+          JBP_CONFIG_SPRING_AUTO_RECONFIGURATION: '{enabled: false}'
+          MAVEN_REMOTEREPOSITORIES[REPO1]_URL: https://repo.spring.io/libs-snapshot
+          SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_CONNECTION_URL: https://api.run.pivotal.io
+          SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_CONNECTION_ORG: <your-cloud-foundry-org>
+          SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_CONNECTION_SPACE: <your-cloud-foundry-space>
+          SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_CONNECTION_DOMAIN: cfapps.io
+          SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_CONNECTION_USERNAME: <your-cloud-foundry-username>
+          SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_CONNECTION_PASSWORD: <your-cloud-foundry-password>
+          SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_CONNECTION_SKIP_SSL_VALIDATION: true
+          SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_CLOUDFOUNDRY_ACCOUNTS[default]_DEPLOYMENT_SERVICES: postgres-service, rabbitmq-service
+          SPRING_CLOUD_SKIPPER_CLIENT_SERVER_URI: <your-skipper-server-uri> # e.g. https://my-skipper-server.cfapps.io/api
+          SPRING_CLOUD_DATAFLOW_SERVER_URI: <your-dataflow-server-uri> # e.g. https://my-data-flow-server.cfapps.io
+          SPRING_DATASOURCE_HIKARI_MINIMUMIDLE: 2
+          SPRING_DATASOURCE_HIKARI_MAXIMUMPOOLSIZE: 4
+          SPRING_CLOUD_DATAFLOW_APPLICATIONPROPERTIES_TASK_SPRING_DATASOURCE_HIKARI_MINIMUMIDLE: 1
+          SPRING_CLOUD_DATAFLOW_APPLICATIONPROPERTIES_TASK_SPRING_DATASOURCE_HIKARI_MAXIMUMPOOLSIZE: 2
+        services:
+          - postgres-service
+    ```
+    For an explanation of the configured properties, see https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#\_common\_application_properties
```

Comment:
Add an empty line before the sentence that starts with "For an". Please preserve its indentation (four spaces), so that it remains a child of the step and not an independent paragraph (which would also break the numbering).

Reply:
There was an extraneous set of backticks.